### ISSUE TITLE
Feedback from Tung

### ISSUE THREAD

**Issue by tungfam** [TheSwiftAlps/feedback-2019#1]
Hello,

Thank you very much for the organisation! It was super nice and useful.

I liked that:
1. It was in a workshop format.
2. I was able to connect and talk to so many cool devs. Usually on the normal conferences you don't really connect that closely to other people.
3. Pairing during the workshop.

What can be improved:
1. Would be nice if we all could live in 1 place. It would be a lot more engaging and fun. I know this was considered, but maybe there is some other way :)
2. Put a marker next to the bottles, so that people can write the name on the bottle/cup and later they reuse it. It's easily to be lost among other same bottles, and people end up taking new bottles before finishing previous one.
3. Have separate rooms for each workshop.
4. Have a "booking" system with priorities for the workshops. Might be hard to do, so this is just an idea.

**Comment by BasThomas** [comment]
First of all: this was an awesome, awesome conference!

I want to add my vote to improvement 3. I was about to open a separate issue for it, but this works as well, I think.

It was great to have Adam's workshop in a separate room, and I think the others can benefit from it as well. I didn't have many issues giving the workshop, but while attending I noticed it was hard to follow the mentors from time to time. :)
